incubator/jenkins/master-image/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM jenkins:2.7.3
```

Comment:
Honest to gosh Im not worried about it. 

I'd prefer things to be more self contained than not when the overhead is small (read handful of lines). As a maintainer I want to watch one set of PRs and issue sets for my application/chart. This slight tweak to the docker image is critical to the functionality of my chart (installing k8s plugin) and is not universally useful if you aren't using my chart.